.github/workflows/test.yml

```diff
@@ -26,4 +26,18 @@ jobs:
         env:
           PERCY_POSTINSTALL_BROWSER: true
       - run: yarn build
+      - name: Set up @percy/cli from git
```

Comment:
make `on:` similar to other repos. 

```yml
on: 
  push:
  workflow_dispatch:
    inputs:
      branch:
        required: false
        type: string
        default: master
```

Reply:
is `L28` - `yarn build` required? I believe `yarn test:coverage` handles building as well?

Reply:
I think we need it

Reply:
can you try it once? since other repos doesn't have it

Reply:
Removed it